content/docs/api/openapi.md

```diff
@@ -0,0 +1,47 @@
+---
+metaTitle: 'Qri OpenAPI'
+metaDescription: 'Documentation for the Qri OpenAPI'
+weight: 1
+
```

Comment:
We haven't finished updating the _actual_ documentation, correct? If so, is there any way to mark this page as a draft until we're done updating the doc? I don't want to publish out of date docs, it'll only make things worse

Reply:
I do not know about the status of the `.yaml` documentation.

There is an issue for errors in the spec. qri-io/qri#1482

But the correctness compared to the actual served API, I do not know.

Note it as as a draft in the text of the page? Or do you mean something else?